internal/hardware/validator.go

```diff
@@ -202,7 +202,7 @@ func isISCSINetworkingValid(disk *models.Disk, inventory *models.Inventory) erro
 	}
 	iSCSIHostIP, err := netip.ParseAddr(disk.Iscsi.HostIPAddress)
 	if err != nil {
-		return fmt.Errorf("Cannot parse iSCSI host IP %s: %w", disk.Iscsi.HostIPAddress, err)
+		return fmt.Errorf("iSCSI IPv4 address is not routable")
```

Comment:
I don't think errors from `ParseAddr` make any claims about if the address is routable.

I think the problem here is that `disk.Iscsi.HostIPAddress` was empty. Perhaps what we need is a separate check specifically for that.

Reply:
Or rather I think it would be good to add this check to the previous condition. So line 199 would be:

```go
if disk.Iscsi == nil || disk.Iscsi.HostIPAddress == "" {
  return fmt.Errorf("Host IP address is not available")
}
```